src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -711,6 +817,11 @@ static void window_loadsave_populate_list(rct_window *w, sint32 includeNewItem,
                 safe_strcat_path(listItem->path, fileInfo.path, sizeof(listItem->path));
                 listItem->type = TYPE_FILE;
                 listItem->date_modified = platform_file_get_modified_time(listItem->path);
+
+                // Cache a human-readable version of the modified date.
+                listItem->date_formatted = Platform::FormatShortDate(listItem->date_modified);
```

Comment:
These assignments seem to be problematic. I'd expect a copy constructor to kick in from the returned std::string, but that doesn't seem to be happening, resulting in instability.